docker-compose-v3.7.yml

```diff
@@ -0,0 +1,74 @@
+version: "3.7"
```

Comment:
maybe we should name this file just `docker-compose.yml` as it's the default, also makes easier to test fwd locally on the project

Reply:
adding this here because a i can't comment on symlink diff:

symlink? doesn't work on Windows, might be a blocker for Windows support

Reply:
so the symlink solves the case for running ./fwd locally while developing it.

In the case of Windows support, I am fine not supporting Windows for `fwd` development - that is, `fwd` can be used in Windows, but for developing/adding to it, you can 1. use Mac or Linux, or 2. Fix the Symlink by copying the file locally.